java/google/registry/keyring/api/PgpHelper.java

```diff
@@ -117,7 +117,7 @@ public static PGPKeyPair lookupKeyPair(
           new BcPBESecretKeyDecryptorBuilder(new BcPGPDigestCalculatorProvider())
               .build(new char[0]));
     } catch (PGPException e) {
-      throw new VerifyException(e.getMessage());
+	throw new VerifyException(e.getMessage(), e);
```

Comment:
Don't log the exception's message and the exception itself as an inner exception; you're duplicating the message that way.  If you don't have anything to add, then do `new VerifyException(e)`, or maybe add a simple error message, like `new VerifyException("Could not look up PGP keypair", e)`

Reply:
Fixed.